packages/jupyter-ai/jupyter_ai/context_providers/_examples.py

```diff
@@ -0,0 +1,131 @@
+# This file is for illustrative purposes
+# It is to be deleted before merging
+from jupyter_ai.models import HumanChatMessage
+from langchain_community.retrievers import ArxivRetriever, WikipediaRetriever
```

Comment:
1. Those aren't specifically coding related so I thought it wouldn't make sense to be default in a coding assistant. Perhaps if it were some stackoverflow or general google retriever that can help with coding questions, it would serve better as defaults.
2. They are incomplete implementations. The text provided for both can either be too long (entire article/wiki page) or too short (their abstract/summary). There is still a lot of work to be done to extract the right amount of context.
3. For my corporate use case, I don't have direct internet connection (use LLMs via proxy) so I'll need a way to disable such defaults.

Reply:
Note that if we do require all context providers to be triggered via commands, then these can be included even if the implementation is a WIP, since the feature will be opt-in instead of always-on. 👀

Reply:
That makes sense; let's just delete `_examples.py` before merging then.